maven-embedder/src/main/java/org/apache/maven/cli/CLIManager.java

```diff
@@ -48,6 +48,8 @@
 
     public static final char VERBOSE = 'X';
 
+    public static final String VERY_VERBOSE = "XX";
```

Comment:
If this is really really required, please rename it to TRACE, IMO that's the common way to call it.

Reply:
@rfscholte That would be against the logic we have deprecated `--debug` with `-X` only.

Reply:
Yes, I see your point, although has "tracing" multiple meanings, like debug does?

Reply:
Correct and we should avoid multiple meanings, no?